src/declarations/contracts/telemetryTypes/telemetryType.ts

```diff
@@ -1,85 +1,3 @@
-export type TelemetryTypeKeys =
```

Comment:
We don't appear to use these `telemetryTypes` anywhere in the project any longer.